### DISCUSSION TITLE
MatPlotLib Not working right

### DISCUSSION THREAD

**Discussion by christian-villandry** [cooperrc/computational-mechanics#189] [Q&A]
Good evening, I was wondering if anyone got this error. I did some digging online and it said to reload my imports and i did that and im still getting it. Its not a huge deal as i dont really need axis titles, but it is kind of annoying.
![image](https://github.com/cooperrc/computational-mechanics/assets/135077694/9537c095-8dee-4cc9-95b8-8e4d9d2fe8fc)

**Comment by cooperrc** [comment]
What? It definitely works for me, 

```python
plt.plot(year, pop/1e9, 'o')
plt.xlabel('year')
plt.ylabel('population (billions)')
```

Maybe restart kernel, then try importing matplotlib again, 

```python
import numpy as np
import matplotlib.pyplot as plt
plt.style.use('fivethirtyeight')
```
